docs/components/buttons/index.html

```diff
@@ -3,6 +3,8 @@
 minver: 0.1.6
 also:
   components/buttons/test.html: Testing - Buttons
+  components/loaders/#busy: Busy Loader
+  components/files/#file-selector: File Selector
   components/menus: Menus
```

Comment:
might want to look into which components to link to -- consult with Chris in the near future